docs/source/contributing/developing-core.md

```diff
@@ -206,19 +216,38 @@ docker volume rm volto-backend-data
 
 Then run `make backend-docker-start` again to start the backend with a clean data volume.
 ````
-    
-Browse to the backend running at http://localhost:8080.
 
-In the second session, start the frontend.
+
+(develop-volto-configure-backend-language-label)=
+
+#### Configure backend language
+
+If you use the Docker image [`plone-backend`](https://github.com/plone/plone-backend), you can set its `LANGUAGE` environment variable, overriding the default of `en`, when you start it.
```

Comment:
For what it's worth, this variable is only applied when the Plone site is created. If you start the backend multiple times without deleting the volume where it persists data, the LANGUAGE is only used the first time.

Reply:
Fixed in 83365fe5477a2f011df2dc6eb5828ed1233b141d. The bonus sentence may be obvious, but explicit yadda yadda...